src/routes/todos/todo-controller.ts

```diff
@@ -21,9 +22,21 @@ export const todoController = {
     }
   },
 
-  getAll: async (req: Request, res: Response<ToDo[]>, next: NextFunction) => {
+  getAll: async (
+    req: Request<
+      {},
+      {},
+      {},
+      { sort?: keyof ToDo; order?: OrderType; search: string }
+    >,
+    res: Response<ToDo[]>,
+    next: NextFunction
+  ) => {
     try {
-      const toDoList = await todoService.listAll();
+      const sortBy = req.query.sort;
+      const order = req.query.order;
+      const searchString = req.query.search;
```

Comment:
You can destructor the req.query object to define this variables in one line
`const { sortBy, order, searchString } = req.query`